app/serializers/participant_serializer.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class ParticipantSerializer
+  include JSONAPI::Serializer
+
+  set_id :id
+  attributes :email, :full_name
+
+  attributes :mentor_id do |user|
+    if user.early_career_teacher?
+      user.early_career_teacher_profile.mentor&.id
+    end
```

Comment:
```suggestion
    user&.early_career_teacher_profile&.mentor&.id
```

The condition essentially does the same thing.